float64 bounds_radius
bool robot_in_bounds
bool fly_in_bounds